### DISCUSSION TITLE
bark-coding.xyz is really cheap

### DISCUSSION THREAD

**Discussion by nmsderp** [cytrinc/bark-coding#172] [Admin General]
https://www.namecheap.com/domains/registration/results/?domain=bark-coding.xyz
![image](https://github.com/mariocraft987/bark.github.io/assets/130254323/fe86e9fd-bd89-489f-a5ba-bab5504f1952)
this is something @mariocraft987 would have to do, but if we did do it a lot of cool things could happen

**Comment by mariocraft987** [comment]
Oh

**Comment by nmsderp** [comment]
this would be better than buying from somewhere else like vercel which is 20$ a year

**Comment by mariocraft987** [comment]
Wait.... vercel hosting has cost to it??????

**Comment by nmsderp** [comment]
no, you can buy domains from vercel, vercel is free, its just an option if you want to get a custom domain, you can use namecheap domains on vercel

**Comment by raynepaws** [comment]
the tld "xyz" might appear as "sketchy" to some people. what about `bark-coding.dev`? still relatively cheap compared to other ones...

_also, i **love** the .dev tld_
![Screenshot from 2024-03-11 16-36-32](https://github.com/mariocraft987/bark.github.io/assets/129226914/e1849966-b7f8-4cfc-8165-fd3a83a9857b)

**Comment by mariocraft987** [comment]
yea

**Comment by Rubiidev-18** [comment]
Would bark-program be better?
https://www.namecheap.com/domains/registration/results/?domain=bark-program.xyz
![Screenshot_20240313_165315985Quick](https://github.com/mariocraft987/bark.github.io/assets/158863401/d0570136-a4fa-496e-9689-2f9f95c921d2)

**Comment by nmsderp** [comment]
bark-program sounds really weird, bark-coding sounds a lot better

**Comment by mariocraft987** [comment]
yes!

**Comment by quirkyworks** [comment]
What about bark-coding.com?

**Comment by raynepaws** [comment]
> What about bark-coding.com?

`.com`s are generally pretty cheap, but we don't make a profit off of this project. `.com`, standing for commercial, meaning money.

**Comment by quirkyworks** [comment]
bark-coding.dev seems good, if there are any other options

**Comment by mariocraft987** [comment]
Yea, I agree

**Comment by nmsderp** [comment]
if anyone is up for it, they should buy bark-coding.dev

**Comment by raynepaws** [comment]
> if anyone is up for it, they should buy bark-coding.dev

if nobody else'll do it, i'll do it. sooner we get it done, the better, because we don't want anybody to snag that domain before us

**Comment by nmsderp** [comment]
> > if anyone is up for it, they should buy bark-coding.dev
> 
> if nobody else'll do it, i'll do it. sooner we get it done, the better, because we don't want anybody to snag that domain before us

i can't really do it, @thepython555 already owns a ton of domains so i don't think we should add on to that money, but maybe @mariocraft987 could do it

**Comment by dumorando** [comment]
> > > if anyone is up for it, they should buy bark-coding.dev
> > 
> > 
> > if nobody else'll do it, i'll do it. sooner we get it done, the better, because we don't want anybody to snag that domain before us
> 
> i can't really do it, @thepython555 already owns a ton of domains so i don't think we should add on to that money, but maybe @mariocraft987 could do it

ill only buy it if bark actually goes somewhere :shrug:

**Comment by mariocraft987** [comment]
That's the use of Donating, so I guess someone or **me** should get a _cash app_ acount

**Comment by nmsderp** [comment]
> That's the use of Donating, so I guess someone or **me** should get a _cash app_ acount

It would make the most sense for you to have it, we probably shouldn't set up donations util this project is actually usable. Which we are getting close to since I got p5.js working.

**Comment by mariocraft987** [comment]
Nice

**Comment by nmsderp** [comment]
Any updates?

**Comment by mariocraft987** [comment]
Not yet, we need a way to 𝚐̶𝚎̶𝚝̶ earn money.

**Comment by raynepaws** [comment]
> Not yet, we need a way to 𝚐̶𝚎̶𝚝̶ earn money.

you could set up a paypal account and show our thing on a donation website like kickstarter or something

**Comment by mariocraft987** [comment]
Ohhh yes!

**Comment by raynepaws** [comment]
> Ohhh yes!

come to think about it, we should've set up a kickstarter quite a while ago
it's actually a really good idea :0

**Comment by mariocraft987** [comment]
I agree!

**Comment by raynepaws** [comment]
> I agree!

whenever possible, definitely get a paypal/cashapp/whatever account so we can get this website running

**Comment by mariocraft987** [comment]
k

**Comment by CharcoalAfterlife** [comment]
> the tld "xyz" might appear as "sketchy" to some people. what about `bark-coding.dev`? still relatively cheap compared to other ones...
> 
> _also, i **love** the .dev tld_ ![Screenshot from 2024-03-11 16-36-32](https://private-user-images.githubusercontent.com/129226914/311894332-e1849966-b7f8-4cfc-8165-fd3a83a9857b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQyODc1MzIsIm5iZiI6MTcxNDI4NzIzMiwicGF0aCI6Ii8xMjkyMjY5MTQvMzExODk0MzMyLWUxODQ5OTY2LWI3ZjgtNGNmYy04MTY1LWZkM2E4M2E5ODU3Yi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNDI4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDQyOFQwNjUzNTJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT01YmNiYmM4ZTFkMDg0NzNmNGU3NDAyZDk4MjAxODJjNGY0YTczOWE3ZDZmNzIwODk1OTdmM2I1NzM0NGVkZjZmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.CyW7qVm4kBYf27jHbw7LFddz186TqQ8-VC1jYddz0p8)

yeah it's cool and all but what about .com, .org?

**Comment by mariocraft987** [comment]
`.com` is a commercial domain while Bark is a nonprofit

**Reply by dumorando** [reply]
my website is dumorando.com and as a human i dont think i am for profit

**Reply by nmsderp** [reply]
> `.com` is a commercial domain while Bark is a nonprofit

after doing some research, you have to be a registered nonprofit to say that because it is illegal in some states and countries, so for you to say that you would need to register it as one

**Comment by nmsderp** [comment]
bump

**Comment by mariocraft987** [comment]
`bark.dev` seems like a good URL, it's common and easily reachable

**Comment by nmsderp** [comment]
bump

**Reply by nmsderp** [reply]
also if you want it could be bark.nmsderp.xyz i own nmsderp.xyz

**Reply by mariocraft987** [reply]
Just as @qwertyy-dev says
> the tld "xyz" might appear as "sketchy"

**Reply by raynepaws** [reply]
`.dev` or `.org` tld's are our best bet

**Reply by mariocraft987** [reply]
I SAY bark.dev

**Reply by nmsderp** [reply]
someone owns that

**Reply by raynepaws** [reply]
bark-coding.dev

**Reply by Rubiidev-18** [reply]
> ye that's good, the redirect can be barkcoding.dev and a few typosquats.
lets do barkcoding.dev

**Comment by Bashamega** [comment]
Hello i think is-a.dev subdomain will be good for now.
Why?
Because, it is free and easy to use.

**Reply by raynepaws** [reply]
but bark is not a dev, not even a person

**Reply by Bashamega** [reply]
> but bark is not a dev, not even a person

Correct. But I thought it can be used for now

**Reply by mariocraft987** [reply]
Go ahead, it will be a temporarly domain

**Reply by nmsderp** [reply]
is-a.dev is only for portfolios or personal websites about a person, you will get rejected

**Reply by Bashamega** [reply]
> is-a.dev is only for portfolios or personal websites about a person, you will get rejected

May be we can send a quick 📧

**Reply by mariocraft987** [reply]
🤨

**Reply by dumorando** [reply]
consider the following: https://github.com/tarampampam/free-domains .is-an.app

**Comment by mariocraft987** [comment]
@nmsderp, what domain did you say that I can transfer?
